Clarify K8s deploy events listener & remove duplicates (#2486)
logzio/logz-docs: _source/logzio_collections/_log-sources/k8s-deploy-events.md

##### Optional Custom Listener

> If you have an HTTP/s endpoint that receives JSON input then by setting the `customListener` secret you can override the Logz.io listener output and route the data to the custom listener instead. 

Reviewer: ```suggestion
If you have a custom Logz.io listener, you can use the parameter `customListener` and override the Logz.io listener parameter and route the data to the custom listener. 
```

> Replace `<<CUSTOM-HOST>>` with your listener endpoint URL. 

Reviewer: ```suggestion
Replace `<<CUSTOM-HOST>>` with your custom endpoint URL. 
```
